tests/unit/other.tcl

```diff
@@ -268,7 +268,6 @@ start_server {overrides {save ""} tags {"other"}} {
         $rd reset
 
         # skip reset ouptut
-        $rd read
```

Comment:
this was reading the `+OK`, right?
i don't think we wanna remove it, just change the order and put it after the next line.
i.e.
```
        assert_equal [$rd read] "RESET"
        assert_equal [$rd read] "OK"
```
i suppose the fact this test currently passes is because reset is affecting the current client monitor flag after the fact, and the RESET command is still logged.
am i missing anything?

Reply:
No, this was reading the monitor response. Since the monitor code executed before the reset command, the client will get the RESET and the get reset.

Reply:
but you did keep the `assert_equal [$rd read] "RESET"` below, and the test passed.

Reply:
ohh, the `resetCommand` replies with `RESET` (not with `OK`).
odd.